### PR TITLE
add rart60

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

Add Rart60 with processor RP2040 and support via

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rart/rart60/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# RART60
+
+![rart60](https://i.imgur.com/8RkCYQE.jpg)
```
Comment (fauxpark) [inline keyboards/rart/rart60/readme.md:3]:
```suggestion
![rart60](https://i.imgur.com/8RkCYQEh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart60/readme.md (hunk 1 above)
Comment (alabahuy) [reply keyboards/rart/rart60/readme.md]: done :) @fauxpark

**Review by fauxpark — APPROVED** [review]

File: keyboards/rart/rart60/config.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 16
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
```
Comment (fauxpark) [inline keyboards/rart/rart60/config.h:31]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rart/rart60/config.h (hunk 2 above)
Comment (alabahuy) [reply keyboards/rart/rart60/config.h]: done :)

**Review by fauxpark — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**